### DISCUSSION TITLE
Multiple image picker

### DISCUSSION THREAD

**Discussion by webfletch** [ionic-team/capacitor#3479] [Ideas]
Are there any plans to add a Capacitor version of the image picker?

https://ionicframework.com/docs/native/image-picker

**Comment by jcesarmobile** [comment]
There are no plans for adding new plugins supported by Ionic

There is a community plugin that allows to pick multiple pictures (only on iOS at the moment) https://github.com/capacitor-community/media

And Capacitor supports Cordova plugins, so you can still use that plugin

**Reply by webfletch** [reply]
My App targets both iOS and Android so the community one would not be of any use at the moment for me. 

I understand that you can use Cordova plugins with Capacitor but from what I understand I would then need both Cordova and Capacitor installed in my App and I cannot see the benefit of using Capacitor over Cordova at the moment when Cordova seems to do everything that is needed. 

I don't think it would make sense instead of using one or the other, to have most things run on Capacitor but still need Cordova for some plugins, it seems like I would be adding unnecessary code/plugins to my App by having them both installed at the same time, unless there is a distinct advantage to using Capacitor for most things over Cordova that I have missed?

**Reply by jcesarmobile** [reply]
No, if you use Capacitor, you use Capacitor, and Capacitor is backward compatible with Cordova plugins, you don’t have to install Cordova to use Cordova plugins in Capacitor.

**Reply by webfletch** [reply]
Okay so I can run through the migration steps and then remove Cordova but then still use Cordova plugins for the ones that aren't available in Capacitor?

Possible a question for another thread or discussion somewhere else but are there any noticeable benefits to using Capacitor rather than Cordova. Are the plugins faster or more reliable than the Cordova equivalent?

**Reply by jcesarmobile** [reply]
Exactly, once you migrate you can uninstall Cordova but still use the cordova plugins you had
